server/src/main/java/io/littlehorse/common/model/repartitioncommand/RepartitionSubCommand.java

```diff
@@ -1,10 +1,10 @@
 package io.littlehorse.common.model.repartitioncommand;
 
-import io.littlehorse.server.streams.store.ModelStore;
+import io.littlehorse.server.streams.stores.TenantScopedStore;
 import org.apache.kafka.streams.processor.api.ProcessorContext;
 
 public interface RepartitionSubCommand {
-    public void process(ModelStore repartitionedStore, ProcessorContext<Void, Void> ctx);
+    public void process(TenantScopedStore repartitionedStore, ProcessorContext<Void, Void> ctx);
```

Comment:
Metrics are stored in a "Cluster" store. how are you planning to do that with this design?

Reply:
Or are you planning to store it into the "default" tenant?